src/System.IO/pkg/System.IO.pkgproj

```diff
@@ -3,7 +3,7 @@
   <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
   <ItemGroup>
     <ProjectReference Include="..\ref\System.IO.csproj">
-      <SupportedFramework>net463;netcoreapp1.1;uap10.1;$(AllXamarinFrameworks)</SupportedFramework>
```

Comment:
I agree, and then you can keep the P2P as well.

---
In reply to: [82022847](https://github.com/dotnet/corefx/pull/12351#discussion_r82022847) [](ancestors = 82022847,82022610)